src/rpcblockchain.cpp

```diff
@@ -492,6 +492,15 @@ Value getblockchaininfo(const Array& params, bool fHelp)
     obj.push_back(Pair("difficulty",            (double)GetDifficulty()));
     obj.push_back(Pair("verificationprogress",  Checkpoints::GuessVerificationProgress(chainActive.Tip())));
     obj.push_back(Pair("chainwork",             chainActive.Tip()->nChainWork.GetHex()));
+    obj.push_back(Pair("autoprune",             fPruneMode));
```

Comment:
I think we're abolishing the name "autoprune". Can you just call it "pruned" or something?